gammapy/background/__init__.py

```diff
@@ -2,6 +2,7 @@
 """
 Background estimation and modeling methods.
 """
+from .energy_offset_array import EnergyOffsetArray
```

Comment:
Change this line to
```python
from .energy_offset_array import *
```

The `*` means to import all names listed in `__all__` in that module (i.e. `EnergyOffsetArray`).
The result will be the same, the difference is that we then maintain the list of public names from the `gammapy/background/energy_offset_array.py` module in that file and not have to remember to also update a list of names here in case more things are added.